upgrading/rosa-upgrading-cluster-prepare.adoc

```diff
@@ -0,0 +1,46 @@
+[id="rosa-updating-cluster-prepare"]
+= Considerations for upgrading
+include::modules/common-attributes.adoc[]
+ifdef::openshift-dedicated,openshift-rosa[]
+include::modules/attributes-openshift-dedicated.adoc[]
+endif::[]
+:context: rosa-updating-cluster-prepare
+
+toc::[]
+
+{product-title} 4.9 uses Kubernetes 1.22, which removed a significant number of deprecated `v1beta1` APIs.
+
+{product-title} 4.8.14 introduced a requirement that an administrator must provide a manual acknowledgment before the cluster can be upgraded from {product-title} 4.8 to 4.9. This is to help prevent issues after upgrading to {product-title} 4.9, where APIs that have been removed are still in use by workloads, tools, or other components running on or interacting with the cluster. Administrators must evaluate their cluster for any APIs in use that will be removed and migrate the affected components to use the appropriate new API version. After this evaluation and migration is complete, the administrator can provide the acknowledgment.
+
+Before you can upgrade your {product-title} 4.8 cluster to 4.9, you must provide the administrator acknowledgment.
+
+[id="rosa-upgrading-requirements_{context}"]
+== Requirements for upgrading your {product-title} clusters
+
+Before you can upgrade your {product-title} clusters, you must update the needed tools to the appropriate version.
+
+include::modules/rosa-upgrading-preparing-4-8-to-4-9.adoc[leveloffset=+2]
+
+include::modules/upgrade-49-acknowledgement.adoc[leveloffset=+3]
```

Comment:
In this module we have the same exact text block as we do above (line 13).  Do we need the same exact text twice?

Reply:
Yup, verbatim the same. Remove the duplication